docs/asl/ref/offset.md

```diff
@@ -1,6 +1,6 @@
 @@@ atlas-signature
 TimeSeriesExpr
-Duration
+Duration or List\[Duration\]
```

Comment:
The list variant is deprecated and not something we want to encourage. Also it has a different signature as it would result in a `StyleExpr` and cannot be used for math operations.

https://github.com/Netflix/atlas/wiki/style-offset

Reply:
Ack. I'll tweak this a bit then. I'd like to leave a note in here since it appears to be in-use and threw me for a loop while parsing.